### PR TITLE
Deployment guidelines - first draft

### PR DESCRIPTION
**Pull request by zeger-pon**

### CODE REVIEW

File: chapters/generic/checklist.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 - License files checked and added to project <<260S>>?
 - All resources in U.S. English <<103>>?
-- <<documentation,Documentation>> available?
+- <<documentation,Documentation>> available for both code an deployment?
```
Comment (bjanssen-pon) [inline chapters/generic/checklist.adoc:6]: I think we also need an architectural document or process-/flowdiagram

File: chapters/generic/documentation.adoc [hunk 2]
```diff
@@ -24,3 +24,9 @@ monitored by automated systems.
 * <<gnu-coding-standards>>, Chapter 5.2
 
 
+[#293] 
+== {MUST} document deployment procedures
+
+Deployment of the tool, program or integration must be documented. This
```
Comment (bjanssen-pon) [inline chapters/generic/documentation.adoc:30]: Also needs to include a list of `recources`. What kind of resources does it use within IE GCP or AWS? (think of serviceaccounts, BigQuery, PubSub, etc)

File: chapters/generic/quality.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
+[[quality]]
+= Quality
+
+[#294]
+== {STATUS_RFP} {SHOULD} have seperate environments for development, testing, acceptance and production
```
Comment (bjanssen-pon) [inline chapters/generic/quality.adoc:5]: The environments should be available before starting development.

File: chapters/generic/checklist.adoc (hunk 1 above)
Comment (bjanssen-pon) [inline chapters/generic/checklist.adoc:6]: Requirement document of the PO?

**Review by bjanssen-pon — COMMENTED** [review]
I left some comments for you to review :)

File: chapters/generic/quality.adoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+In order to maintain software or integration quality the setup of an DTAP
+environment is preferred. This setup allows for an atomized way of working on
+features and bug fixes.
+
+This environment consists of four parts:
```
Comment (denverweij) [inline chapters/generic/quality.adoc:11]: only three are described. Production is missing. Production can only be updated after a sign of

**Pull request merged**